Mit.md

```diff
@@ -0,0 +1,10 @@
+## Copyright 2023, [Chandan Gupta,Samuel Lemma]
+
+###### Please delete this line and the next one
+###### APP TYPE can be a webpage/website, a web app, a software and so on
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] It would be nice if you remove these two lines to complete your License file well :+1: and to make your project licensed properly :blush: